### ISSUE TITLE
Can't control my units

### ISSUE THREAD

**Issue by JacobItaly** [EldarFara/VictorianEraDev#5]
Hello!

Whenever after some time I just can't control my units. Controlling them by UI in U doesn't work, and also doesn't work with Backspace or just simply f1 (or f1-f3 to charge, they all just standing there). Is that because the mod from ModDB is outdated, and where should I look for the newest version? If you ask, yes I used all of the hotkeys and followed the prompts shown in the chat. Please help, it will be greatly appreciated.

**Comment by JacobItaly** [comment]
UPDATE: installed the version that's provided here on Github, same problem.. Please help, because this is great mod and i've spent alot of time playing it and having fun.

**Comment by EldarFara** [comment]
select units first

**Comment by JacobItaly** [comment]
Well, funny thing is - I can't select them. Or i just don't understand what is your point. I tried selecting them multiple times with open panel on "U key" (clicking, etc) and still no results. The tilde key to appearantly "choose" too doesn't work either. Tried click / dragging - same thing. Is there another key combination for it? (i'm not everyday Warband player, just playing it few years just for fun, just to let you know)

ps.: If you ask me, yeah i've read all of the prompts multiple times, that told me what to do. I've also selected units before the battle begun - no succesfull results. We are also playing for fun, with cheats sometimes - we used to do that all the time in Native Warband - to have better start, etc, or to just have fun & cause wars, or import / export character to blast everybody and i'm wondering if cheats affected negatively onto the modification?
